docs/rc.xml.all

```diff
@@ -589,4 +589,27 @@
   <menu>
     <ignoreButtonReleasePeriod>250</ignoreButtonReleasePeriod>
   </menu>
+
+  <!--
+    Magnifier settings
+
+    'width' sets the width in pixels of the magnifier window.
+    'height' sets the height in pixels of the magnifier window.
+    'initScale' sets the initial magnification factor at boot.
+    'borderColour' is an XML RGB string with a leading # which sets the
+      colour of the border of the magnifier window.
+    'borderWidth' sets the width in pixels of the border of the
+      magnifier window.
+    'useFilter' sets whether to use a bilinear filter on the magnified
+      output or simply to take nearest pixel.
+  -->
+    <magnifier>
+        <width>400</width>
+        <height>400</height>
+        <initScale>2</initScale>
+        <borderColour>#ff0000</borderColour>
+        <borderWidth>1</borderWidth>
```

Comment:
I think these two belong to the theme, then we also don't need to add the rgb parser in `rcxml.c`.

Reply:
Also, please use the american spelling of `Color` for the setting, it matches the rest of the codebase.

Reply:
Will do - didn't know what standard was being used here...

Reply:
Should "colour" be used in documentation as well, or just in code?